### PR TITLE
Update backup-and-restore.md

### PR DESCRIPTION
**Pull request by tr0njavolta**

### CODE REVIEW

File: content/spaces/backup-and-restore.md [hunk 1]
```diff
@@ -53,7 +65,7 @@ spec:
         key: creds
 ```
 
-Then create a managed control plane with reference to the `SharedBackupConfig`.
+Then create two managed control planes with references to the `SharedBackupConfig`.
```
Comment (phisco) [inline content/spaces/backup-and-restore.md:68]:
```suggestion
Then create two managed control planes.
```

File: content/spaces/backup-and-restore.md [hunk 2]
```diff
@@ -146,17 +161,49 @@ kubectl delete controlplane my-awesome-ctp
 
 ### Restore
 
+
 <!-- vale off -->
-To restore from a backup, check the `ControlPlane` to make sure it's ready.
+To restore from a backup, create a new control plane with the `sharedBackupConfigRef` or `BackupConfigRef` you created.
```
Comment (phisco) [inline content/spaces/backup-and-restore.md:166]: there is no sharedBackupConfigRef anymore, this should say only something like:
```
You can restore a controlplane's state from a backup as follows.
```

**Comment by phisco** [comment]
you can also drop:
```

**Comment by phisco** [comment]
Damn, shift-enter closing PRs hit me again, it's illegal, I have to find a workaround to prevent that from happening...
I wanted to say that you can drop the following part, as it's now part of the restore scenario a few line belows (`kubectl delete controlplane my-awesome-ctp`):


Once your Space indicates the backup is complete, you can delete the managed control plane:
```bash
kubectl wait backup my-awesome-ctp-backup --for condition=Completed=True --timeout=3600s && \
kubectl delete controlplane my-awesome-ctp
```

File: content/spaces/backup-and-restore.md (hunk 2 above)
Comment (tr0njavolta) [inline content/spaces/backup-and-restore.md:166]:
```suggestion
You can restore a control plane's state from a backup in a few steps.
```

File: content/spaces/backup-and-restore.md [hunk 3]
```diff
@@ -133,30 +145,58 @@ metadata:
...
 ### Restore
 
+
 <!-- vale off -->
-To restore from a backup, check the `ControlPlane` to make sure it's ready.
+You can restore a control plane's state from a backup in a few steps.
 <!-- vale on -->
 
+
+First, create a Secret in the source control plane:
```
Comment (phisco) [inline content/spaces/backup-and-restore.md:163]: I would make it more explicit this is just to have some dummy data to backup and restore, it's not a required step.

**Pull request merged**